test/services/sync/strategies/CSVSyncer/CSVSyncer.unit.test.js

```diff
@@ -75,5 +75,35 @@ describe('CSVSyncer', () => {
 			expect(result[1].class).to.equal('');
 			expect(result[2].class).to.equal('2b');
 		});
+
+		it('should not accept falsy valus as dates for user birthday', () => {
+			const falsyValues = [false, null, undefined, '', 0];
+			const out = new CSVSyncer();
+			const { errors } = out.stats
+			falsyValues.forEach((f) => out.assertDateFormat(f));
+			errors.forEach((r) => expect(r.message).to.be.equal('Kein Wert für das Geburtstagsdatum'))
+		});
+		it('should not accept strange values as dates for user birthday', () => {
+			const out = new CSVSyncer();
+			const { errors } = out.stats
+			const misfitValues = [Symbol('42'), [], {}, true];
+			misfitValues.forEach((m) => out.assertDateFormat(m));
+			errors.forEach((r) => expect(r.message).to.be.equal('Falscher Wert für Datum. Muss String sein'))
+		});
+		it('should not accept invalid date format as user birthday', () => {
+			const out = new CSVSyncer();
+			const { errors } = out.stats
+			const wrongFormatDates = [
+				'32.12.2000', '01.13.2000', '01.01-2000', '01/01.2000', '01-01/2000', '42', 'void'];
+			wrongFormatDates.forEach((w) => out.assertDateFormat(w));
+			errors.forEach((r) => expect(r.message).to.be.equal('Falsches Format. Muss sein dd.mm.yyyy oder dd/mm/yyyy oder dd-mm-yyyy'))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 137. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5220139)